new-pipeline/src/dist.js

```diff
@@ -1163,12 +1168,12 @@ function displaySingleHistogramSet(axes, useTable, histograms, title,
       max_y: maxPercentage + 2, // Add some extra space to account for the top label
       transition_on_update: false,
       target: axes,
-      x_label: histogram.description,
```

Comment:
The description was overlapping the labels. Plus we already show the description above the histogram, the one below was superfluous.